editoast/src/chartos/layer_cache.rs

```diff
@@ -0,0 +1,103 @@
+use core::f64::consts::PI;
+
+use super::BoundingBox;
+
+// File containing dead_code until https://github.com/DGEXSolutions/osrd/issues/1795 is done
+
+#[allow(dead_code)]
+pub struct Tile {
+    x: i64,
+    y: i64,
+    z: i64,
+}
+
+#[allow(dead_code)]
+fn xy_from_latitude_longitude(latitude: f64, longitude: f64, zoom: i64) -> (i64, i64) {
```

Comment:
You have some issues with your types here. Most of the `i64` should be `u64`.

```suggestion
fn xy_from_latitude_longitude(latitude: f64, longitude: f64, zoom: u64) -> (u64, u64) {
```